src/encoded/tests/data/inserts/gene.json

```diff
@@ -172,7 +172,7 @@
 }, {
     "geneid": "2995",
     "uuid": "83a311d1-7ed3-41bb-85f0-1f00c458e6fc",
-    "dbxrefs": ["MIM:110750", "HGNC:4704", "ENSEMBL:ENSG00000136732", "Vega:OTTHUMG00000131464", "UniProtKB:P04921"],
+    "dbxrefs": ["MIM:110750", "HGNC:4704", "ENSEMBL:ENSG00000136732", "Vega:OTTHUMG00000131464", "UniProtKB:P04921", "RefSeq:NG_007479"],
```

Comment:
I don't think these changes should be in the PR?

Reply:
yea, this looks funny.  The commit history has a merge commit, ENCD-4216, and ENCD-4208.